貢獻者詞條項目重覆回傳bug fix

aweimeow opened the pull request:
在這裡因為可能貢獻者對於一個外語貢獻多種說法，
因說法不同故仍需計算，

如果對於某詞條，貢獻者提供 A & B 兩種說法，
這樣子應該要計算成貢獻條目 = 2

_但是在貢獻者頁會出現兩次外語條目_，此 PR 修復了這個問題

Commit:
done test for 貢獻者詞條重覆回傳

Commit:
忘記把詞條貢獻 test 增加到 2 (詳細見說明)

在這裡因為可能貢獻者對於一個外語貢獻多種說法，
因說法不同故仍需計算，

如果對於某詞條，貢獻者提供 A & B 兩種說法，
這樣子應該要計算成貢獻條目 = 2

Commit:
修完 bug !

sih4sing5hong5 commented on 臺灣言語平臺/介面/貢獻者.py:
```diff
@@ -27,7 +27,8 @@ def 貢獻者表(request):
     for user in map(lambda x: x[1], sorted(contributor_dict.items(),
```
`dict`有`items`、`keys`、`values`三種

aweimeow replied:
這我當初想很久想不到好的寫法 XD
只好以自己的認知寫出目前能寫出最好的 code 了 QQ
想問問如果是你的話，會怎麼寫這一段 code 呢？

sih4sing5hong5 replied:
用`values`就好呀，就免`map`矣

Commit:
Update sorted statement

Commit:
fix coding style

貢獻者.py:27:5: E265 block comment should start with '# '
貢獻者.py:30:17: E128 continuation line under-indented for visual indent
